bin/util.js

```diff
@@ -101,6 +145,57 @@ module.exports = {
         }, {});
     },
 
+    /**
+     * Extract curl options in the provided options.
+     * Create curl  command from the options.
+     *
+     * @param {Object} options - Commander.Command Instance
+     * @param {String} url - Requested url
+     * @returns {String} - Curl command
+     */
+    createCurl: (options, url) => {
```

Comment:
@raghavbhatia336 Let's identify if there could be a way to not create a tight coupling with curl options because it will affect our ability to support new options. Since we are just taking the options from the user and passing them onto the next module, we shouldn't really handle the options.
Can you please look into whether `program.args` solve our problem? This variable provides the user's command in an array form.